Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:alpine
+MAINTAINER Tecnativa <info@tecnativa.com>
+ENTRYPOINT ["/usr/src/app/entrypoint.sh"]
+ENV GIT_AUTHOR_NAME=git-aggregator \
+    EMAIL=https://hub.docker.com/r/tecnativa/git-aggregator
```

Comment:
Once you set up your automated build, you'd have to refer to `https://hub.docker.com/r/acsone/git-aggregator` here.